python/pyMaCh3/__init__.py

```diff
@@ -0,0 +1,3 @@
+from ._pyMaCh3 import __doc__, fitter, manager, plotting, sample_pdf, splines
+
+__all__ = ["__doc__", "fitter", "manager", "plotting", "sample_pdf", "splines"]
```

Comment:
With the other fixes this can be left blank and it'll work!

Reply:
it seems like this is still needed, but the other python module files (manager.py, fitter.py ...) aren't any more so I've removed them